### ISSUE TITLE
Missing translations - Polish language

### ISSUE THREAD

**Issue by TitanSpirit** [owncloud/music#712]
I would like to offer translation from English to Polish, how can I help?

**Comment by paulijar** [comment]
Hello, and thanks for the offer. Translations for this app are created in Transifex within the owncloud-org project. In principle, you should be able to contribute translations if you navigate to https://www.transifex.com/signin/?next=/owncloud-org/, log in (you can use your github account), and join the Polish translation team.

However, we are currently having an integration issue with Transifex: only a small portion of all the localizable strings of the Music app show up in Transifex. All the strings currently available on Transifex already seem to have a Polish translation. The mentioned integration issue is tracked at https://central.owncloud.org/t/owncloud-music-app-translations/14881.

**Comment by paulijar** [comment]
As the Transifex problem has not been moving anywhere, I guess we should forget it for now. Instead, see the following post for an offline process to contribute translations: https://github.com/owncloud/music/issues/671#issuecomment-782746463

**Comment by p-bo** [comment]
@paulijar Regarding migration to Nextcloud Transifex - just open issue on https://github.com/nextcloud/docker-ci/issues/new/choose (choose Request translations). And you will get checklist as for example this: https://github.com/nextcloud/docker-ci/issues/426 and you can ask [rakekniven](https://github.com/rakekniven) if something is unclear to you.

**Comment by p-bo** [comment]
For motivation - this developer [was very impressed](https://github.com/powerpaul17/nc_money/issues/4#issuecomment-1274590428) how much work can be done by Nextcloud translators community :-)

**Comment by paulijar** [comment]
@p-bo Thanks for the hint. This looks interesting; I need to look into it closer once I have time.

**Comment by paulijar** [comment]
@rakekniven If I would do as suggested above and registered this app to be translated using Nextcloud Transifex, then would that necessarily mean that the bot would automatically use the Nextcloud translation tool to scrape the translatable strings and update the .pot file? Because that would almost certainly be the wrong way for this project and it would find only a tiny fraction of all the translatable strings. That's basically the reason why we had to disable the ownCloud Transifex integration. Most of our translations don't use functions `t()` or `n()` but are implemented using angular-gettext.

If, however, it would be possible for me to update the .pot file manually (using suitable scripts, of course), and the bot would update only the .po files, then this could be a solution. Alternatively, it could work if there was a way to pass a custom command for the bot to extract the translatable strings.

**Comment by paulijar** [comment]
Is there any chances that one day in the future it could be more configurable? I imagine that the bot works in phases something like
1. Pull the latest changes from github
2. Scrape the code to regenerate .pot
3. Upload the .pot to Transifex
4. Download all the translated .po files from Transifex
5. Push all the updated files to github

Now, if we could opt out from the phase 2 and keep the rest, then everything would be wonderful.

**Comment by paulijar** [comment]
Good question. At least I haven't completely given up on this, but I haven't actively pursued it, either. But one day, I hope I have the time and inspiration to look into this.

**Comment by paulijar** [comment]
@rakekniven Not really. On 15.4.2025 you wrote that you would reply my email about this matter after your vacation, so presumably you have been having a really long vacation since I'm still waiting :D. But seriously speaking, I did figure out you probably forgot that mail but I haven't done any further investigation since I've had my hands full with other things.
